gammapy/makers/utils.py

```diff
@@ -536,3 +539,103 @@ def make_counts_off_rad_max(geom_off, rad_max, events):
         counts_off.fill_events(selected_events)
 
     return counts_off
+
+
+def make_observation_time_map(observations, geom, offset_max=None):
+    """
+    Compute the total observation time on the target geom
+    for a list of observations
+
+    Parameters
+    ----------
+    observations : `~gammapy.data.Observations`
+            Observations container containing list of observations.
+    geom : `~gammapy.maps.Geom`
+            Reference geom.
+    offset_max : `~astropy.units.quantity.Quantity`
+        The maximum offset FoV. If None, it will be taken from the IRFs
```

Comment:
```suggestion
    offset_max : `~astropy.units.quantity.Quantity`, optional
        The maximum offset FoV. Default is None.
        If None, it will be taken from the IRFs.
```